nnpdf_data/theory.py

```diff
@@ -0,0 +1,59 @@
+"""
+This module provides an unique source and definition for all the possible parameters
+that a theory card can contain. 
+It also implement some utilities function to manage the theory cards. 
+"""
+
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class _TheoryCard:
+    ID: int
+    PTO: int
+    FNS: str
+    DAMP: int
+    DAMPPOWERc: None #Power of the damping factor in FONLL for the c
+    DAMPPOWERb: None #Power of the damping factor in FONLL for the b
+    IC: int
+    IB: int
+    ModEv: str
+    ModSV: None#
```

Comment:
This is actually used in `pineko` ([here](https://github.com/NNPDF/pineko/blob/4d1e3184038573b452c7addb48fd36ef85cafc7c/src/pineko/evolve.py#L26)).